dotnet3.5/S3/CopyObjectExample/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
+
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace CopyObject
+{
+    // This example uses Amazon Simple Storage Service (Amazon S3) to copy
+    // an object from one Amazon S3 bucket to another. It uses the AWS SDK for .NET 3.5
+    // and .NET 5.0
```

Comment:
All code comments must be syntactically and grammatically complete -- including ending periods -- here and throughout.

Reply:
Checked comments to make sure they are grammatically correct and use complete sentences only.